example/jupyter/tutorial_xgboost.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# XGBoost on SQLFlow Tutorial\n",
+    "\n",
+    "This is a tutorial on train/predict XGBoost model in SQLFLow, you can find the design doc from [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/xgboost_on_sqlflow_design.md), in this tutorial you will know how to:\n",
```

Comment:
Instead of linking the design doc, linking the user guide might be more useful.

Reply:
done